### DISCUSSION TITLE
Copy all config from one device to another

### DISCUSSION THREAD

**Discussion by speedf1** [cagnulein/qdomyos-zwift#2565] [Q&A]
I want to be able to copy the complete settings/profiles/everything from one Android phone to another.  I've tried using the backup/restore function within the app but this doesn't seem to work (and seems to be specific to the profile currently selected?).  I've also tried copying all files from the documents/qz folder from one phone to another, but this doesn't work either.  How can I get an exact copy of the setup duplicated?

**Comment by cagnulein** [comment]
from the settings page, use the save icon and simply copy the file on the
new one and use the opposite icon to restore it

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno mar 3 set 2024 alle 21:51 speedf1 ***@***.***> ha
scritto:

**Comment by karan** [comment]
> simply copy the file

Where is the file stored on Android? `Android/data` isn't accessible by any file managers and I don't get a prompt during backup to choose a target folder. Kinda makes it hard to copy the file.

**Reply by cagnulein** [reply]
No use the icon in qz as I said to restore them, not copy using the files app

**Reply by karan** [reply]
Well I'm changing devices so I need to move the files somehow.

**Comment by karan** [comment]
.

**Comment by cagnulein** [comment]
Yes with the export button you can save them for example in the download
folder and in the new one do the viceversa

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno lun 14 apr 2025 alle 06:10 Karan Goel ***@***.***>
ha scritto:

**Reply by karan** [reply]
Hmm I don't get that. Like I said, it just saves the file and I get a message saying to look in the private folder.

**Comment by cagnulein** [comment]
If you are on android < 13 enable the android document setting in the
experimental settings so qz will save them under /documents/qz and so you
can copy them easily

Remember instead that in order to load them you need to use the icon, don't
move files directly in that folder

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno lun 14 apr 2025 alle 06:25 Karan Goel ***@***.***>
ha scritto:

**Reply by karan** [reply]
Thank you. That worked fine. But restore doesn't seem to work now - I select the file and it says settings have been loaded correctly, however after restarting QZ the settings don't seem to be restored.

**Reply by cagnulein** [reply]
Did you NOT move the file in the qz folder of the new phone?

If so are you able to share the settings file so I can try?

**Reply by karan** [reply]
No I placed it in the Documents folder (and enabled the experiment you mentioned above), then selected the file during restore. Is that not what it's supposed to be?

**Comment by cagnulein** [comment]
Documents folder or qz under document folder? If you put it on the qz
folder you have to delete it. You can't place file there, due to Google
security stuff

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno sab 19 apr 2025 alle 09:16 Karan Goel ***@***.***>
ha scritto:

**Reply by karan** [reply]
Yes I put it in the Documents folder directly.

**Comment by cagnulein** [comment]
/documents  or /documents/qz directly?

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno sab 19 apr 2025 alle 17:35 Karan Goel ***@***.***>
ha scritto:

**Reply by karan** [reply]
`/documents`

**Comment by cagnulein** [comment]
So it should be fine. Send me over the file and I will check it

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno sab 19 apr 2025 alle 17:42 Karan Goel ***@***.***>
ha scritto:

**Reply by karan** [reply]
Sure here you go. https://send.bitwarden.com/#gIGgcRQDTk-MYrLEAWpAJA/m8BpID9fOPfPdkcJPEnLyQ (I removed my username/password from it of course).

**Reply by karan** [reply]
@cagnulein Any ideas how I can restore my settings? I had a lot of custom settings and it'd be nice if I didn't have to redo them all again.

**Reply by cagnulein** [reply]
Hi @karan I tried to download it but I got this message

The Send you are trying to access does not exist or is no longer available.

Please upload it again
Thanks

**Reply by karan** [reply]
Sure here's another copy: https://send.bitwarden.com/#dLAigwsUXEOSKLLHAWomoA/7Q0BWN3sOcVldCcfKyyhjw

**Reply by cagnulein** [reply]
Tried and it worked fine here https://github.com/user-attachments/assets/9dd0dad9-b642-4735-8df0-e0b2f8aaf93c



I checked the weight and the age for example and they are yours right?

**Reply by karan** [reply]
I'm not sure but I mainly want/expect the tile positions to be restored too. I don't think that's happening for either of us.

**Comment by cagnulein** [comment]
I didn't check them, i will do ASAP

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno mer 23 apr 2025 alle 21:12 Karan Goel ***@***.***>
ha scritto:

**Reply by karan** [reply]
Yeah the FTP tiles shouldn't be there for example. My old phone has a pretty basic setup with a handful of tiles.

**Reply by cagnulein** [reply]
Is this right? ![Screenshot_20250423-212943.png](https://github.com/user-attachments/assets/6311d422-2eb1-4282-a598-2e72ff7f618a)

I simply restarted qz after loading

**Comment by cagnulein** [comment]
Ok, I'll check it out.


Il giorno mer 23 apr 2025 alle 21:18 Karan Goel ***@***.***>
ha scritto:
